certified-connectors/CobbleStone - Contract Insight/apiDefinition.swagger.json

```diff
@@ -0,0 +1,965 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2.0",
+    "title": "CobbleStone Contract Insight",
+    "description": "The Cobblestone - Contract Insight API connector allows you to connect our contract management software with many popular web and data services. The basic out of the box (OOB) endpoints will allow you to review Schema table names and whether they can be searched, edited, and updated. You can pull Entity information on table names, and Query, add, and update Customer record data.",
+    "contact": {
+      "name": "Matthew Friebis",
```

Comment:
@nizarhandal can we please change it to something like `CobbleStone team`? This way it won't be so personalized, names are usually used for independent publishers.